cli/module_loader.rs

```diff
@@ -458,12 +469,50 @@ impl CliModuleLoader {
       return Err(generic_error("Attempted to load JSON module without specifying \"type\": \"json\" attribute in the import statement."));
     }
 
+    let code_cache = if module_type == ModuleType::JavaScript {
```

Comment:
Just to double check - this check means that "transpiled" source is JS, ie. currently only JSON modules are not using the cache?

Reply:
Correct